README_.md

```diff
@@ -0,0 +1,177 @@
+# 1 BladeDISC
```

Comment:
I'm not sure do we need the **number** on the title, the suggestion layout in https://google.github.io/styleguide/docguide/style.html#document-layout is as the following:

``` text
# Document Title

Short introduction.

[TOC]

## Topic

Content.

## See also

* https://link-to-more-info
```

Reply:
done